flake.nix

```diff
@@ -0,0 +1,81 @@
+{
+  description = "Mantis flake";
+
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
+    nixpkgs.url =
+      "github:nixos/nixpkgs?rev=a98302aa9b9628915878a6ea9776c40a0bb02950";
+    nixpkgs-sbt = {
+      url = "github:nixos/nixpkgs?rev=e2bb73ce5f786b83e984b80199112f86b8a6cc9d";
+      flake = false;
+    };
+    sbtix = {
+      url =
+        "github:input-output-hk/Sbtix?rev=7b969a5641fce10500ca51cbe88af4ea160d7064";
+      flake = false;
+    };
+  };
+
+  outputs = { self, nixpkgs, flake-utils, nixpkgs-sbt, ... }@inputs:
+    let
+      overlay = final: prev: {
+        inherit (import nixpkgs-sbt { inherit (final) system; }) sbt;
+        sbtix = final.callPackage ./sbtix.nix { };
```

Comment:
I would not use sbtix, but instead sbt-derivation. Mantis is already using it on the `develop` branch